frontend/src/components/Header/Header.tsx

```diff
@@ -721,8 +727,8 @@ const BillingBanner: React.FC = () => {
 	])
 
 	const isMaintenance = moment().isBetween(
-		'2023-03-05T23:30:00Z',
-		'2023-03-06T01:45:00Z',
+		systemData?.system_configuration?.maintenance_start,
+		systemData?.system_configuration?.maintenance_end,
 	)
```

Comment:
checked that `moment().isBetween(undefined, undefined)` returns `false` so we should be ok if no data is loaded